odk1-src/form-widgets.rst

```diff
@@ -925,104 +760,304 @@ XLSForm Rows
+
+.. image:: /img/form-widgets/geotrace5.*
+  :alt: The same map as displayed previously, but now a series of red markers form a line across the map.
+
+.. image:: /img/form-widgets/geotrace6.*
+  :alt: The same map as previously, with a new modal popup. The headline of the modal is "Save GeoTrace as" followed by two options: Save as Polygon and Save as Polyline. In the bottom-right is a Cancel button.
+
+.. image:: /img/form-widgets/geotrace7.*
+  :alt: The Geotrace form widget, as shown previously. The question text is "Where have you been?" and the button label is "View or Change GeoTrace." Below that is a list of lat/long GPS coordinates.
 
-The default :ref:`geopoint` widget does not display a map to the user. With the added :tc:`maps` appearance attribute, a map of the recorded location is shown to the user. The user cannot select a different location on the map. (See :ref:`placement-map-widget` for a geopoint with a user-selected location.)
 
-XLSForm Rows
-""""""""""""""
+.. rubric:: XLSForm
 
 .. csv-table:: survey
-  :header: type, name, label, appearance, hint
+  :header: type, name, label
 
-  geopoint,geopoint_widget_maps,Geopoint widget,maps,geopoint type with maps appearance
+  geotrace, trace_example, Where have you been?
+  
+  
+.. _geoshape-widget:
 
-.. _default-geoshape-geotrace:
+Geoshape
+~~~~~~~~~
 
-Geoshape and Geotrace
-~~~~~~~~~~~~~~~~~~~~~~
+type
+  :tc:`geoshape`
+appearance
+  *none*
+
+Captures a user-entered series of GPS coordinates, forming a polygon.
+
+.. image:: /img/form-widgets/geoshape-start.*
+  :alt: The GeoShape form widget, as displayed in the ODK Collect app on an Android phone. The question text is "Select an Area." Below that is a button labeled "Start GeoShape."
+
+.. image:: /img/form-widgets/geoshape1.*
+  :alt: A modal popup over a map. The modal headline is "Zoom to..." Below that are two options: "Zoom to current location" (selected) and "Zoom to saved feature". In the bottom-right of the modal is a Cancel button.
+
+.. image:: /img/form-widgets/geoshape2.*
+  :alt: A map displayed in the ODK Collect App on an Android phone. Above the map is the instruction: "Long press to place marks." On the right side are five icon buttons stacked vertically: Add marker, Zoom, Layers, Trash, Save.
+
+.. image:: /img/form-widgets/geoshape3.*
+  :alt: The same map as displayed previously, but now a series of red markers form a polygon across the map.
+
+.. image:: /img/form-widgets/geoshape4.*
+  :alt: The GeoShape form widget shown previously. The question text is "Select an Area." The button label is now "View or Change GeoShape." Below the button is a list of lat/long GPS coordinates.
+
+.. rubric:: XLSForm
 
-The :ref:`geopoint` and :ref:`geotrace` widgets do not accept any appearance attributes.
+.. csv-table:: survey
+  :header: type, name, label
 
-.. _single-select-widgets:
+  geoshape, shape_example, Select an area
+  
+  
+  
+.. _single-select-widget:
 
-Single Select Widgets
+Single select widget
```

Comment:
Wondering whether it might be helpful to have a short introduction to selects with the text you have in the "Including Images as Choices" section below all the select widgets. Then it gives a good sense of the possibilities.

Reply:
that makes sense. I'll try to make that work.